### DISCUSSION TITLE
Terms of Use and Privacy Statement

### DISCUSSION THREAD

**Discussion by Tettsch** [leihs/leihs#1267] [Q&A / Community Support] [answered]
Hello,
we would like to set up leihs at our university department but we have to add Terms of Use and a Privacy Statement. Is there a possibility to do that at the bottom of each page in leihs? 

#### Expected behaviour
- Modify the links in the footer within the Admin Panel

#### Specifications
- Leihs version: 6.1.1

**Comment by eins78** [comment, marked as answer]
There are no settings to configure the the bottom links. We currently have no plans to implement this but would accept pull request.

For "Terms of Use" there is a setting to show a link to some externally hosts "lending terms" and requires acceptance by ticking a checkbox _for each order_. Maybe this an option for you. For details on how to use it, see this feature spec: https://github.com/leihs/leihs-integration-tests/blob/8a7e757c176c3ef1c86239855b2d5052de708cb6/spec/features/settings/lending_terms_acceptance.feature

**Comment by winampdevil** [comment]
@Tettsch
maybe it's resolved in the meantime, but...
same thing here in our university department. 
In fact, you can add links to Privacy Statement and Terms of Use by editing the file `/leihs/legacy/app/views/application/_footer.html.haml ` at your server.

i.e. by adding 

```
  impressum_link = 'url-to-impressum'
  impressum_hint = 'Impressum'
  impressum_name = 'Impressum'

  privacy_link = 'url-to-privacy'
  privacy_hint = 'Datenschutzerklärung'
  privacy_name = 'Datenschutzerklärung'
  
    .col2of10.padding-top-xs.padding-left-xs
      %a.headline-m.light{href: impressum_link, title: impressum_hint}
        = impressum_name


    .col2of10.padding-top-xs.padding-left-xs
      %a.headline-m.light{href: privacy_link, title: privacy_hint}
        = privacy_name
```



So you should have the links on every page - unfortunately this doesn't work for the startpage.
And i think, you must restart the leihs.legacy-service and also re-editing this file after every update of leihs.

edit:
i'm using this in leihs 6.2.4
